app/components/Snaps/SnapUIRenderer/components/image.ts

```diff
@@ -12,7 +12,8 @@ function generateBorderRadius(
     case 'medium':
       return 6;
 
-    // TODO: Support Full border radius (50%)
+    case 'full':
+      return 100;
```

Comment:
```suggestion
      return "50%";
```

Maybe something like this to indicate that this is not a pixel value?

Reply:
I'm fine with `number | 'full'` too!